docs/conditions.md

```diff
@@ -44,6 +44,13 @@ type ApplicationStatusCondition struct {
 }
 ```
 
+## Library
+We're going to use a shared library to provide the condition types to operators.
+This will ensure the code is not specific to any operator and it will allow
+products outside of CNV to also consume it.
+
+https://github.com/openshift-kni/operator-status
```

Comment:
@markmc WDYT? We talked to operator-sdk about owning this, but they don't want to commit to it until they get past some other priorities. All operators in the CNV and OCS ecosystems will need to share these conditions, so openshift-kni seems like a reasonable org to be their home for now.

Reply:
I see openshift-kni as downstream of openshift, kubevirt, etc. - suggest doing this under github.com/openshift